.solcover.js

```diff
@@ -9,6 +9,12 @@ module.exports = {
         'zeppelin/ERC20.sol',
         'zeppelin/ERC20Basic.sol',
         'zeppelin/Ownable.sol',
-    ],
-    copyNodeModules: true,
+        'test/ens/ENS.sol',
```

Comment:
We can actually just use a directory name here for all the ignored files, as solidity-coverage will automatically detect that we want to ignore the entire directory.